kjourney/src/app/view1/view1.component.ts

```diff
@@ -0,0 +1,260 @@
+import {Component, OnInit, ViewEncapsulation} from '@angular/core';
```

Comment:
this whole approach needs a rethink! but for now leave a comment in the code to look into this again!